include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge_sort.h

```diff
@@ -272,17 +274,17 @@ struct __merge_sort_global_submitter<_IndexT, __internal::__optional_kernel_name
                             const oneapi::dpl::__ranges::drop_view_simple __rng2(__dst, __offset + __n1);
 
                             const auto start = __find_start_point(__rng1, __rng2, __i_elem_local, __n1, __n2, __comp);
-                            __serial_merge(__rng1, __rng2, __rng /*__rng3*/, start.first, start.second, __i_elem,
-                                           __chunk, __n1, __n2, __comp);
+                            __serial_merge<__gpu_chunk>(__rng1, __rng2, __rng /*__rng3*/, start.first, start.second,
```

Comment:
I think renaming `__gpu_chunk` to something like `__unroll_factor` is more clear since there is a different `__chunk` and the device may not be a GPU.

Reply:
Semantically in this code the variable exactly mean the value of `chunk` for any device that can also be used to accelerate a 3D graphics API. In other words - the chunk for non-CPU device.